data/AnushaSP07.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Anusha SP",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Application Developer. Enthusiastic Learner. Content Writer",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/AnushaSP07/",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@anushasp07",
+      "icon": "FaMedium"
+    },
+    {
+      "name": "Follow me on HashNode",
+      "url": "https://hashnode.com/@AnushaSP07",
+      "icon": "FaHashnode"
+    },
+    {
+      "name": "Connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/anushasp07/",
+      "icon": "FaLinkedIN"
```

Comment:
Also, change the `FaLinkedIN` to `SiLinkedin`
``` suggestion
      "icon": "SiLinkedin"
```

Reply:
Thank you @priyankarpal , sure I will make changes and commit